climakitae/explore/warming.py

```diff
@@ -234,7 +237,9 @@ def get_sliced_data(y, level, years, window=15, anom="Yes"):
 
         if anom == "Yes":
             # Find the anomaly
-            anom_val = y.sel(time=slice("1981", "2010")).mean("time")
+            anom_val = y.sel(time=slice("1980", "2010")).mean(
```

Comment:
Why is 1981 changed to 1980 here?

Reply:
It was originally 1980 when written by Naomi. I think in some WL code from before I accidentally changed it to 1981, and so I just changed it back to what it was before. I can also move this to a separate PR if wanted/needed.

Reply:
1981-2010 is actually the standard climatological period, so I'd recommend using that. But leave it for now and return to it later!

Reply:
Put a trello card up about this for both of us